{{ cookiecutter.app_name }}/pyproject.toml

```diff
@@ -14,7 +14,7 @@ icon = "src/{{ cookiecutter.module_name }}/resources/{{ cookiecutter.app_name }}
 sources = ['src/{{ cookiecutter.module_name }}']
 requires = [
 {%- if cookiecutter.gui_framework == 'PySide2' %}
-    'pyside2==5.13.0',
+    'pyside2==5.15.2',
```

Comment:
Is there any reason that we shouldn't use `>=` rather than `==` here? It seems like the class of problem we've seen  on this occasion would be fixed with a floating version specifier; the end user can then lock down that version specifier once it's been generated.

Reply:
That's a good question. Since qt5 and qt6 use different pip packages it does make sense using `>=`. On the other hand, can we get rid of the hardcoded version number to use the latest available version?

Reply:
Removing the version specifier entirely is certainly an option. However, I would argue that since we know there are some versions (in the 5.14 range?) that don't work, it's worth having a minimum bound. This ensures that (a) we don't accidentally end up with a bad version because a local virtual environment has an old version installed, and (b) provides some light documentation of a specific version that we've verified as working.

Reply:
Aha, great. I agree. Changed the setting to `>=`